File: EJERCICIOS EN CLASE/CodigoCondicional/codigo_Condicional.cpp
#include <iostream>
using namespace std;

int main() {
    int num;

    cout << "Ingrese un numero: ";
    cin >> num;

    if (num > 0 && num < 100) {
        cout << "Es positivo y menor a 100" << endl;
    } else {
        cout << "No es positivo o es mayor a 100" << endl;
    }

    return 0;
}

File: EJERCICIOS EN CLASE/CodigoCondicional/codigo_Condicional.java
import java.util.Scanner;

public class codigo_Condicional {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            int num;

            System.out.println("Ingrese un numero");

            num = sc.nextInt();

            if (num > 0 && num < 100) {
                System.out.println("Es postivo y menor a 100");
            } else {
                System.out.println("No es postivo o es mayor a 100: ");
            }
        }
    } 
}
